vocabularies/Org.OData.JSON.V1.xml

```diff
@@ -64,7 +64,7 @@
         </Collection>
       </Annotation>
 
-      <Term Name="Schema" Type="JSON.JSON" Nullable="false" AppliesTo="Property">
+      <Term Name="Schema" Type="JSON.JSON" Nullable="false" AppliesTo="Parameter Property ReturnType Term TypeDefinition">
```

Comment:
Can a term have type `Edm.Stream`? [OData-CSDL, sec. 14.3](https://docs.oasis-open.org/odata/odata-csdl-json/v4.01/odata-csdl-json-v4.01.html#sec_ConstantExpression) does not tell how they can be specified.

Reply:
Yes, there is no restriction on the types a term can have, and Edm.Stream is just a primitive type.

See https://issues.oasis-open.org/browse/ODATA-1323 how to specify a stream value in an annotation.

Reply:
The solution for ODATA-1323 does not propose a new subsection of OData-CSDL, sec. 14.3. Isn't that warranted?

Reply:
Good point, added as a comment to ODATA-1323.

Reply:
I would rather add it to the proposal text. Based on my experiences there are chances that it will get lost otherwise.

Reply:
Done